i18n/locales/source/blockly-mooc/common.json

```diff
@@ -1178,7 +1176,7 @@
   "parentLetterStudentPrivacy": "Code.org's commitment to student privacy",
   "parentLetterStudentPrivacyDetails": "Code.org assigns utmost importance to student safety and security. Code.org has signed the [Student Privacy Pledge]({pledgeLink}) and their privacy practices have received [one of the highest overall scores from Common Sense Media]({commonSenseLink}). You can find further details by viewing Code.org's [Privacy Policy]({privacyPolicyLink}).",
   "parentLetterWhy": "Why computer science",
-  "parentLetterWhyDetails": "Computer science teaches students critical thinking, problem solving, and digital citizenship, and benefits all students, no matter what opportunities they pursue in the future. And learning to make interactive animations, code-art, games, and apps on Code.org encourages creativity and makes learning fun.",
+  "parentLetterWhyDetails": "[Six different studies show]({researchLink}): children who study computer science perform better in other subjects, excel at problem solving, and are 17% more likely to attend college. Computer science teaches students critical thinking, problem solving, and digital citizenship, and benefits all students, no matter what opportunities they pursue in the future. And learning to make interactive animations, code-art, games, and apps on Code.org encourages creativity and makes learning fun.",
```

Comment:
confirmed that adding this interpolation variable doesn't break things in the apps i18n engine